test/ui-test/initializationUITest.ts

```diff
@@ -9,12 +11,16 @@ import { getNotifications, cleanUpNotifications } from './common/testUtils';
 export function knativeInitializationUITest(): void {
   let driver: WebDriver;
 
-  before(async () => {
+  before(() => {
     driver = VSBrowser.instance.driver;
-    await cleanUpNotifications();
   });
 
   describe('Knative view', () => {
+    before(async function setup() {
+      this.timeout(10000);
+      await cleanUpNotifications();
```

Comment:
Why did you need to separate this? Both `before` functions get called before the test. Now you just have to repeat it every time. OR do you ONLY need it before this test?

Reply:
I was desperate, test was failing inside NotificationCenter in the lib. and I could not reproduce, very unstable. OS moving out and there. I will clear that up.